app/styles/_settings.scss

```diff
@@ -17,6 +17,7 @@ $wrap-desktop: 115px;
 // Font sizes
 $font-size-tiny: 10px;
 $font-size-x-small: 12px;
+$font-size-xs-small: 13px;
```

Comment:
Give us a couple of days and we'll do a refactor..  😁  Anyway, that size is the UI Kit and spoken with Dani.